e2e/tests/api/helpers/delete.ts

```diff
@@ -0,0 +1,30 @@
+import type { AxiosInstance } from "axios";
+
+import { logger } from "../../common/constants";
+
+export async function deleteSboms(axios: AxiosInstance, sbomIds: string[]) {
+  var existingSbomIds = [];
+
+  for (const sbomId of sbomIds) {
+    try {
+      await axios.get(`/api/v2/sbom/${sbomId}`);
+      existingSbomIds.push(sbomId);
+    } catch (_error) {
+      logger.info(`SBOM with ID ${sbomId} does not exist anymore. Skipping.`);
+    }
+  }
+
+  const deletionPromises = existingSbomIds.map((sbomId) =>
+    axios.delete(`/api/v2/sbom/${sbomId}`).catch((error) => {
+      logger.error(`Failed to delete SBOM with ID: ${sbomId}`, error);
+    }),
```

Comment:
As this is a helper function, helpers should not handle errors but let the caller of the function to deal with the exceptions. Handling the catch here means that if it fails we will only see a log but the function will succeed giving the false impression that the function was executed correctly.